app/forms/openid_connect_token_form.rb

```diff
@@ -113,13 +113,25 @@ def validate_client_assertion
   end
 
   def validate_aud_claim(payload)
-    normalized_aud = payload['aud'].to_s.chomp('/')
-    return if api_openid_connect_token_url == normalized_aud
+    aud_claim = payload['aud']
```

Comment:
Not a blocker, but if you haven't checked it out may be worth looking int using `[Array.wrap](https://apidock.com/rails/Array/wrap/class)` here.